gen/safeq

```diff
@@ -62,10 +62,10 @@ binmode FILE,":utf8";
 ## costcenterno: 1000001
 ## costcentername: Human Resources
 foreach my $id (sort keys %costcentersById) {
-	print FILE "dn: costcenterno=", $id, ", ou=costcenters, o=ysoftsafeq", "\n";
+	print FILE "dn: ", escapeDnValue("costcenterno=", $id, ", ou=costcenters, o=ysoftsafeq"), "\n";
```

Comment:
Be aware, that ',' is used as parameter delimiter, if you want to concat string use '.' operator instead.